backend/src/main/java/it/chalmers/gamma/bootstrap/BootstrapConfig.java

```diff
@@ -0,0 +1,93 @@
+package it.chalmers.gamma.bootstrap;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component("cfg")
```

Comment:
Name it more descriptive, BootstrapConfig or so

Reply:
If there's only one, no need to name it. :thinking: 
Should probably be @Config instead of @Component for clarity